lib/use_cases/save_items_details.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class SaveItemsDetails
+  def initialize(items_gateway:)
+    @items_gateway = items_gateway
+  end
+
+  def execute(items_details:)
+    @items_details = items_details
+
+    errors = []
```

Comment:
I'm not sure what this section is doing? Could be worth extracting it to a method to make it clear what is happening